File: Easy/JavaScript/2144. Minimum Cost of Buying Candies With Discount.js
var minimumCost = function(cost) {
    cost.sort((a, b) => a - b);
    let minCost = 0;
    let n = cost.length;

    for(let i = 0; i < n; i++){
        if(i % 3 != n % 3)
            minCost += cost[i];
    }

    return minCost;
};

File: Easy/Python/2144. Minimum Cost of Buying Candies With Discount.py
class Solution(object):
    def minimumCost(self, cost):
        cost.sort()
        minCost = 0
        n = len(cost)

        for i in range(n):
            if i % 3 != n % 3:
                minCost += cost[i]
        
        return minCost
